tests/integration_test.rs

```diff
@@ -15,6 +19,26 @@ mod integration_test {
         nixci::logging::setup_logging(true);
     }
 
+    #[tokio::test]
+    /// Run `nixci build` and check if the stdout consists of only /nix/store/* paths
+    async fn nixci_build_output() -> anyhow::Result<()> {
+        let mut cmd = Command::cargo_bin("nixci")?;
```

Comment:
After this PR, could you open a new PR using this for the rest of the tests in this file (which current looks up `nixci` in $PATH)?